kategory-docs/docs/docs/datatypes/sequencekw/README.md

```diff
@@ -6,4 +6,61 @@ permalink: /docs/datatypes/sequencekw/
 
 ## SequenceKW
 
-TODO
\ No newline at end of file
+SequenceKW implements lazy lists representing lazily-evaluated ordered sequence of homogenous values.
+
+It can be created from Kotlin Sequence type with a convenient `k()` function.
+
+```kotlin:ank
+import kategory.*
+
+sequenceOf(1, 2, 3).k()
+```
+
+SequenceKW derives many useful typeclasses. For instance, it has a [`Semigroup`](/docs/typeclasses/semigroup/) instance.
+
+```kotlin:ank
+val hello = sequenceOf('h', 'e', 'l', 'l', 'o').k()
+val commaSpace = sequenceOf(',', ' ').k()
+val world = sequenceOf('w', 'o', 'r', 'l', 'd').k()
+
+hello.combineK(commaSpace.combineK(world)).toList() == hello.combineK(commaSpace).combineK(world).toList()
+// ^ returns true because Semigroup is associative
+```
+
+[`Functor`](/docs/typeclasses/functor/)
+
+Transforming a sequence:
+```kotlin:ank
+val fibonacci = generateSequence(0 to 1) { it.second to it.first + it.second }.map { it.first }.k()
+fibonacci.map { it * 2 }.takeWhile { it < 10 }.toList()
+// [0, 2, 2, 4, 6]
+```
+
+[`Applicative`](/docs/typeclasses/applicative/)
+
+Applying a sequence of functions to a sequence:
+```kotlin:ank
+SequenceKW.applicative().ap(sequenceOf(1, 2, 3).k(), sequenceOf({ x: Int -> x + 1}, { x: Int -> x * 2}).k()).toList()
+// [2, 3, 4, 2, 4, 6]
```

Comment:
let `ank` print the values on it's own, otherwise they will show up printed twice, just need to remove the commented outputs.